docs/components/avatar.md

```diff
@@ -9,32 +9,35 @@ Avatar can get props that will define the appearance and behavior of the compone
 
 - **text** - Required. The text that would be rendered, or be used as alt-text for the img.
 - **src** - Optional. The source for the image to be rendered.
-- **size** - Optional. Defaults to `SIZE.LARGE`. Can be one of the sizes (`SIZE`) defined on the component.
-- **shape** - Optional. Defaults to `SHAPE.ROUND`. Can be one of the shapes (`SHAPE`) defined on the component.
+- **size** - Optional. Defaults to `AVATAR_SIZE.LARGE`. Can be one of the sizes (`AVATAR_SIZE`) defined on the component.
+- **tooltip** - Optional. Defaults to empty string. in case empty or not supplied by consumer of the component, `text` prop would be shown as tooltip text
+- **hideTooltip** - Optional. Defaults to `false`.
+- **square** - Optional. Defaults to `false`. Square shape instead of default rounded circle.
+- **border** - Optional. Defaults to `false`. A gentle secondary border.
```

Comment:
You can use it in JSDoc in the component and it will appear in the component props table